docs/summary_env_new.md

```diff
@@ -0,0 +1,49 @@
+# This is the process for creating a functional summary environment
+
+# First create a conda environment named 'jupyter':
+`conda env create -f conda_env_files/jupyter.yaml'
+
+# Then activate the environment and run MMED's setup.py script:
+`conda activate jupyter`
+`python setup.py install`
+
+# With R version 3.5.1, run the following:
+
+R
+install.packages("GGally",dependencies=TRUE)
+install.packages("ggrepel",dependencies=TRUE)
+install.packages("RColorBrewer")
+install.packages("ggplot2")
+
+quit()
+
+# Once all that is installed create a jupyter kernel from the current environment:
+
+`python -m ipykernel install --user --name jupyter --display-name="Jupyter"`
+
+### Install latex to run pdflatex command:
+# on minerva
+module load texlive/2018
+
+# Fedora
+`sudo dnf install texlive texlive-revtex texlive-braket`
+
+# Ubuntu
+`sudo apt install texlive-latex texlive-latex-recommended`
```

Comment:
This seems to be specific for these OS. What if we have some other distro or OS X? Consider how to best organize this.

Reply:
yeah if I have time, I'm going to try again installing texlive-core which is an anaconda package and not OS specific. it works on minerva, on a back version, but haven't gotten it fully working locally yet.